gc/base/standard/Scavenger.cpp

```diff
@@ -5065,3 +5077,16 @@ MM_Scavenger::completeConcurrentCycle(MM_EnvironmentBase *env)
 #endif /* OMR_GC_CONCURRENT_SCAVENGER */
 
 #endif /* OMR_GC_MODRON_SCAVENGER */
+
+#if defined(OMR_ENV_DATA64) && !defined(OMR_GC_COMPRESSED_POINTERS)
+void
+MM_Scavenger::scavenger_poisonSlots(MM_EnvironmentBase *env)
+{
+	_cli->scavenger_poisonSlots(env);
```

Comment:
put a comment that this will poison root slots only (heap access verification is only supported with flat heap configurations).
again, similar comment with scavenge heal